docs/types/reference-types.rst

```diff
@@ -550,3 +550,9 @@ members of the local variable actually write to the state.
 Of course, you can also directly access the members of the struct without
 assigning it to a local variable, as in
 ``campaigns[campaignID].amount = 0``.
+
+.. note::
+    In previous solidity versions, assigments which create memory-structs that contain a mapping
+    like ``campaigns[campaignID] = Campaign(beneficiary, goal, 0, 0)`` were possible,
+    because the compiler would omit the types, in this case the mapping, which are not 
```

Comment:
There's a bit of trailing whitespace here and [the `chk_coding_style` CI job does not like that](https://app.circleci.com/pipelines/github/ethereum/solidity/10828/workflows/0717982b-b997-44b7-9ece-ee1e37cbcaf3/jobs/524074) :)

```suggestion
    because the compiler would omit the types, in this case the mapping, which are not
```

BTW, most text editors have a setting to automatically strip the trailing whitespace. You should enable it.

Reply:
Noted - in theory I have enabled this for my editor.